core/state/sync_test.go

```diff
@@ -204,7 +204,7 @@ func testIterativeStateSync(t *testing.T, count int, commit bool, bypath bool) {
 			codeResults = make([]trie.CodeSyncResult, len(codeElements))
 		)
 		for i, element := range codeElements {
-			data, err := srcDb.ContractCode(common.Hash{}, element.code)
+			data, err := srcDb.ContractCode(common.Address{}, element.code)
```

Comment:
We must roll out a new snap sync protocol for verkle, **later**.

In original scheme, contract code is only identified by code hash. While in verkle, contract code is identified by both address and some other information. Therefore, snap protocol has to be changed.

We can just skip it since verkle doesn't support snap sync anyway.